conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/NegationRuleSuite.scala

```diff
@@ -92,8 +101,13 @@ class NegationRuleSuite extends FunSuite with SparkTestBase with LoggerTestBase{
     testRule(inputDataset, expectedDataset, expectedJSON)
   }
 
-  private def testRule(inputDataset: Dataset[String], enceladusDataset: ConfDataset, expectedJSON: String): Unit = {
-    val inputDf = spark.read.schema(NegationRuleSamples.schema).json(inputDataset)
+  private def testRule(inputDataset: Dataset[String], enceladusDataset: ConfDataset, expectedJSON: String, nullableSchema: Boolean = true): Unit = {
```

Comment:
My personal taste is for `nullableSchema` not to have a default value, but require tests explicitly specify if they want to deal with a nullable or not nullable schema.

Reply:
OK. I was lazy 😉